### DISCUSSION TITLE
Help to compile iped-viewers-impl

### DISCUSSION THREAD

**Discussion by dafeol** [sepinf-inc/IPED#755] [Q&A] [answered]
Boa tarde! Como estão?
Gostaria do auxilio dos colegas para configuração do IPED através do GIT.
Já tentei diversas soluções, incluindo a orientação de instalar o liberica jdk 8 FULL, porém não conseguir solucionar o erro:

![image](https://user-images.githubusercontent.com/7400768/132997825-863eb754-6d96-4af4-a44e-cf93adae34f0.png)

![image](https://user-images.githubusercontent.com/7400768/132997891-f25c48dc-cd77-4846-8013-aa20dd28ba56.png)

![image](https://user-images.githubusercontent.com/7400768/132997920-b036d2dd-eecd-4b11-8145-00827fd99b78.png)

![image](https://user-images.githubusercontent.com/7400768/132997931-2143b746-7749-4818-a3dd-85546e9466fe.png)

Desde já, agradeço pela atenção!

**Comment by lfcnassif** [comment, marked as answer]
There are a lot of java versions in your PATH, remove them all except for the Liberica installation. And also try to point the JAVA_HOME to the liberica installation folder.

**Reply by dafeol** [reply]
Muito Obrigado! As alterações corrigiram o problema.
![image](https://user-images.githubusercontent.com/7400768/133011013-19dca315-aa73-4223-b5ea-32a88d94107a.png)

Aproveito para parabenizá-lo pelo trabalho realizado e excepcional ferramenta criada! 
Comecei a usá-la recentemente e já tem sido muito útil na otimização do serviço. 
Quero muito aprofundar no uso para conhecer as demais funcionalidades.

Sucesso!

**Reply by eandradi2** [reply]
Estou com o mesmo problema, poderia compartilhar as configurações.

**Reply by eandradi2** [reply]
![image](https://user-images.githubusercontent.com/94645443/142865321-871d8147-b476-48a0-b627-3279128a2646.png)

**Reply by lfcnassif** [reply]
Do you have JavaFx installed? This is pointed out in this project first page.

**Reply by eandradi2** [reply]
tanks, sucess

**Comment by lfcnassif** [comment]
Also check mvn -v
